### PR TITLE
Update default RPCGasCap variable 

### PR DESCRIPTION
**Pull request by pro100skm**
# Proposed changes
Describe the big picture of your changes here to communicate to the maintainers why we should accept this pull request.

## Types of changes

What types of changes does your code introduce to XDC network?
_Put an `✅` in the boxes that apply_

- [✅] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation Update (if none of the other choices apply)
- [ ] Regular KTLO or any of the maintaince work. e.g code style
- [ ] CICD Improvement

## Impacted Components
Which part of the codebase this PR will touch base on,

_Put an `✅` in the boxes that apply_

- [ ] Consensus
- [ ] Account
- [ ] Network
- [✅] Geth
- [ ] Smart Contract
- [ ] External components
- [ ] Not sure (Please specify below)

## Checklist
_Put an `✅` in the boxes once you have confirmed below actions (or provide reasons on not doing so) that_

- [ ] This PR has sufficient test coverage (unit/integration test) OR I have provided reason in the PR description for not having test coverage
- [ ] Provide an end-to-end test plan in the PR description on how to manually test it on the devnet/testnet.
- [✅] Tested the backwards compatibility.
- [✅] Tested with XDC nodes running this version co-exist with those running the previous version.
- [ ] Relevant documentation has been updated as part of this PR
- [ ] N/A

### CODE REVIEW

**Comment by pro100skm** [comment]
When we get quotes (calculate swap routes through different liquidity pools/pairs for our DEX) we make large multicall requests to the node that use a lot of gas. We can decrease multicall chunks but it will produce hundreds of separate calls and will overload the net. 

Our current RPC node -`p_static` works with 150m gas limit and gives us better performance. 

I've tried to add special flag to override default value, but I got errors and reverted commit. Please pay attention to it, maybe adding configurable flag will be better solution

**Comment by GalaxySciTech** [comment]
> When we get quotes (calculate swap routes through different liquidity pools/pairs for our DEX) we make large multicall requests to the node that use a lot of gas. We can decrease multicall chunks but it will produce hundreds of separate calls and will overload the net.
> 
> Our current RPC node -`p_static` works with 150m gas limit and gives us better performance.
> 
> I've tried to add special flag to override default value, but I got errors and reverted commit. Please pay attention to it, maybe adding configurable flag will be better solution

i think  the cap should multipleby 50  i seems you only multipleby 5

**Comment by GalaxySciTech** [comment]
> When we get quotes (calculate swap routes through different liquidity pools/pairs for our DEX) we make large multicall requests to the node that use a lot of gas. We can decrease multicall chunks but it will produce hundreds of separate calls and will overload the net.
> 
> Our current RPC node -`p_static` works with 150m gas limit and gives us better performance.
> 
> I've tried to add special flag to override default value, but I got errors and reverted commit. Please pay attention to it, maybe adding configurable flag will be better solution

cloud u show one of your large multicall requests example pls

**Comment by gzliudan** [comment]
Why are there some commits unrelated to gas cap ?

File: params/config.go [hunk 1]
```diff
@@ -48,6 +48,14 @@ var (
 			TimeoutPeriod:        30,
 			MinePeriod:           2,
 		},
+		2000: {
```
Comment (GalaxySciTech) [inline params/config.go:51]: this should delete pls

File: eth/ethconfig/config.go [hunk 2]
```diff
@@ -69,7 +69,7 @@ var Defaults = Config{
 	GasPrice:           big.NewInt(0.25 * params.Shannon),
 
 	TxPool:      core.DefaultTxPoolConfig,
-	RPCGasCap:   25000000,
+	RPCGasCap:   150000000,
```
Comment (gzliudan) [inline eth/ethconfig/config.go:72]: 150000000 is too big. This value is 50000000 in geth.

Comment (pro100skm) [reply eth/ethconfig/config.go:72]: might be. thats why I'm asking to add a special flag to give other node providers possibility to configure it by themselves

Comment (gzliudan) [reply eth/ethconfig/config.go:72]: Yes, flag is a good idea. Please search `RPCGasCap` and `RPCGlobalGasCapFlag` in geth.

**Pull request closed without merging**